src/local/butler/lint.py

```diff
@@ -185,6 +185,14 @@ def execute(_):
   for file_path in py_changed_file_paths:
     _execute_command_and_track_error('pylint ' + file_path)
     _execute_command_and_track_error('yapf -d ' + file_path)
+    futurize_output = _execute_command_and_track_error(
+        'futurize -0 -x libfuturize.fixes.fix_unicode_keep_u '
+        '-x libfuturize.fixes.fix_future_builtins -x lib2to3.fixes.fix_dict '
+        '-x lib2to3.fixes.fix_ws_comma -x lib2to3.fixes.fix_idioms ' +
```

Comment:
Why fix_ws_comma ?

Reply:
idioms and ws_comma are disabled by default and not explicitly disabling them makes the tool print an extra log line for each. Just did it to save some spam here. I'll leave a comment explaining that when addressing the first comment.